src/blueprints/tv2/tv2-action-service.ts

```diff
@@ -16,23 +16,30 @@ import { Tv2GraphicsActionFactory } from './action-factories/tv2-graphics-action
 import {
   Tv2VideoMixerConfigurationActionFactory
 } from './action-factories/tv2-video-mixer-configuration-action-factory'
+import { Tv2VideoClipActionFactory } from './action-factories/tv2-video-clip-action-factory'
+import { Tv2ActionManifestData, Tv2VideoClipData } from './value-objects/tv2-video-clip-data'
+import { PieceType } from '../../model/enums/piece-type'
 
-export class Tv2ActionsService implements BlueprintGenerateActions {
+export class Tv2ActionService implements BlueprintGenerateActions {
   constructor(
     private readonly cameraActionFactory: Tv2CameraActionFactory,
     private readonly transitionActionFactory: Tv2TransitionActionFactory,
     private readonly audioActionFactory: Tv2AudioActionFactory,
     private readonly graphicsActionFactory: Tv2GraphicsActionFactory,
+    private readonly serverActionFactory: Tv2VideoClipActionFactory,
```

Comment:
Mismatch between arguement name and type.

Reply:
Good catch.
I started out naming everything `server` until I remembered we decided to not call it that anymore. This one missed the refactor.